Source/WebCore/rendering/updating/RenderTreeUpdaterGeneratedContent.cpp

```diff
@@ -318,9 +318,15 @@ void RenderTreeUpdater::GeneratedContent::updateWritingSuggestionsRenderer(Rende
 
     auto textWithoutSuggestion = nodeBeforeWritingSuggestionsTextRenderer->text();
 
+#if PLATFORM(MAC)
```

Comment:
If iOS is the special case, I think we should use that rather than `PLATFORM(MAC)`.